### PR TITLE
add support for images

### PR DESCRIPTION
**Pull request by alr2413**
Add support for images in notifications.
Ref: https://github.com/brozot/Laravel-FCM/pull/185

### CODE REVIEW

**Comment by alr2413** [comment]
Ping @williamdes.
Please merge it.
Thanks.

File: src/Message/PayloadNotificationBuilder.php [hunk 1]
```diff
@@ -170,6 +177,21 @@ public function setIcon($icon)
         return $this;
     }
 
+    /**
+     * Indicates the image that can be displayed in the notification
+     * Supports an url or internal image.
+     *
+     * @param $image
```
Comment (williamdes) [inline src/Message/PayloadNotificationBuilder.php]:
```suggestion
     * @param null|string $image
```

**Review by williamdes — APPROVED** [review]

**Comment by williamdes** [comment]
https://github.com/code-lts/Laravel-FCM/commit/a00ae769fc13a1772871b2041367f2d0ab583432.patch

You GIT seems misconfigured `From: alireza setayesh <alirezasetayesh@alirezas-MacBook-Air.local>`

**Comment by alr2413** [comment]
@williamdes.
I fixed that. Check it again.
Thanks.

**Review by williamdes — APPROVED** [review]
💯
I will merge it as soon as I have my laptop open

**Comment by alr2413** [comment]
> 💯
> I will merge it as soon as I have my laptop open

Thanks.

**Pull request merged**